app/views/smart_answers/landing.html.erb

```diff
@@ -1,4 +1,11 @@
 <% start_node = @presenter.start_node %>
+<%
+  start_href = if @presenter.use_session?
+                 session_flow_path(@presenter.name, node_name: @presenter.flow.questions.first.name)
+               else
+                 smart_answer_path(@presenter.name, started: "y")
+               end
```

Comment:
If you go with the suggestion of a `SessionFlowPresenter` above - could this be a method on that class?  If not, maybe a helper method?

Reply:
Or at the least could we move this to a method in the presenter? Just trying to keep logic out of views if possible.

Reply:
I agree - I left the code there as that seems to be a pattern used a lot across GDS apps.

I'll move it to the `CurrentQuestionHelper` as there are already a lot of methods defining paths by presenter, there, and nothing like it in `FlowPresenter`.